src/framework/global/internal/platform/win/wininteractivehelper.h

```diff
@@ -33,7 +32,7 @@ namespace muse {
 class WinInteractiveHelper : public async::Asyncable
 {
 public:
-    static async::Promise<Ret> openApp(const Uri& uri);
+    static async::Promise<Ret> openAppToOpenUri(const Uri& uri);
```

Comment:
I think the previous name was better :)
We don't need information on how to open the app in the name of method, we just need to open the app

Reply:
Actually I think `openUri` could be an even better name, because this method does not just open an app, it opens an Uri _within_ an app. For example, a specific page within Muse Hub, or whatever page in whatever app. And if the app is not installed, the OS will still open the Uri, but in the web browser (which most likely fails to open it, but still).

Reply:
Anyway, I archived that commit now.